pojo.mustache

```diff
@@ -99,40 +117,53 @@ public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}}{{^parent}}
     return this;
   }
   {{/isMap}}
+  {{! end feature: fluent setter methods }}
+  {{! begin feature: getter and setter }}
 
   /**
   {{#description}}
-   * {{{description}}}
+   * {{{.}}}
   {{/description}}
   {{^description}}
    * Get {{name}}
   {{/description}}
   {{#minimum}}
-   * minimum: {{minimum}}
+   * minimum: {{.}}
   {{/minimum}}
   {{#maximum}}
-   * maximum: {{maximum}}
+   * maximum: {{.}}
   {{/maximum}}
    * @return {{name}}
   */
   {{#vendorExtensions.x-java-getter-annotations}}
   {{{.}}}
   {{/vendorExtensions.x-java-getter-annotations}}
- {{#vendorExtensions.x-extra-annotation}}
```

Comment:
It looks like the the upstream already added the ability to place Java annotations using the `x-extra-annotation` vendor extension. We should probably use what's in the upstream.

Reply:
I thought of this but didn't want to break any already existing code that maybe out there.  Plus if you look here I'm not exactly sure that's what they intended:
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java#L1259

It looks as though if we had a `decimal` type and `serializeBigDecimalAsString` is true it would overwrite what was placed in the spec.  I'm gonna test this out and see

Reply:
Looks like you are correct.  Those annotations see to do what we expect as far as I can tell.  Therefore I removed our annotations